examples/show-toast/src/views/App.test.tsx

```diff
@@ -0,0 +1,12 @@
+import {render, getMockContextProps} from "@stripe/ui-extension-sdk/testing";
+import {ContextView} from "@stripe/ui-extension-sdk/ui";
+
+import App from "./App";
+
+describe("App", () => {
+  it("renders ContextView", () => {
+    const {wrapper} = render(<App {...getMockContextProps()} />);
+
+    expect(wrapper.find(ContextView)).toContainText("save to reload this view");
+  });
+});
```

Comment:
```suggestion
```

Reply:
We should either just get rid of the test or provide meaningful ones. I'm ok with the former but not opposed to the latter.

_I'm not sure if committing this suggestion will actually fully remove the file._